### PR TITLE
Increased max depth for hierarchy

### PR DESCRIPTION
**Pull request by himanshu219**

### CODE REVIEW

**Review by mfiglus — COMMENTED** [review]
lgtm from hierarchies side

**Comment by sumovishal** [comment]
@himanshu219 can you update the CHANGELOG.md?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 FEATURES:
 * **New Resource:** sumologic_cse_entity_normalization_configuration (GH-430)
 
+ENHANCEMENTS:
+* Updated maxdepth level for hierarchy resource.
+
 
 ## 2.18.2 (September 1, 2022)
 
-BUG FIXES:
```
Comment (sumovishal) [inline CHANGELOG.md:8]: Bring back this CHANGELOG?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,11 +2,12 @@
 FEATURES:
 * **New Resource:** sumologic_cse_entity_normalization_configuration (GH-430)
 
+ENHANCEMENTS:
+* Updated maxdepth level for hierarchy resource.
```
Comment (sumovishal) [inline CHANGELOG.md:6]: You can mention the PR too (GH-433)

**Review by sumovishal — APPROVED** [review]

**Pull request merged**